hyperdome/client/hyperdome_client.py

```diff
@@ -275,6 +278,7 @@ def server_switcher(self):
         if self.is_connected:
             self.disconnect_chat()
         if self.server_dropdown.currentIndex() == self.server_dropdown.count() - 1:
+            self.__log.debug("adding new server")
```

Comment:
Super pedantic, but the capitalization in the logging bothers me.

Not worth fixing now, but probably sometime in the long-term future.

Reply:
made an issue for it since it's not the first time you've brought up capitalization or the first time I've forgotten to do it